### PR TITLE
story(TG-412): fix style issues

### PR DESCRIPTION
**Pull request by piton13**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/198" target="_blank">ReviewNinja

File: src/app/home/map-view/map-view.component.ts [hunk 1]
```diff
@@ -10,6 +10,6 @@ export class MapViewComponent {
     @Input() currentMapView: MapViewInfoDto;
 
     getImage() {
-        return 'http://localhost:3000/api/files/' + this.currentMapView.pictureName;
+        return `${ENV_PUBLIC_CONFIG.backEndUrl}/api/files/` + this.currentMapView.pictureName;
```
Comment (borys-ihnatyev) [inline src/app/home/map-view/map-view.component.ts]: Use already implemented ```MapViewService.resolvePictureUrl(this.currentMapView)```

**Review by borys-ihnatyev — CHANGES_REQUESTED** [review]

File: src/app/home/map-view/map-view.component.ts (hunk 1 above)
Comment (piton13) [inline src/app/home/map-view/map-view.component.ts]: !fixed

Comment (piton13) [reply src/app/home/map-view/map-view.component.ts]: !fixed.

File: src/app/home/map-view/map-view.component.ts [hunk 2]
```diff
@@ -9,7 +10,9 @@ import { MapViewInfoDto } from './map-view.dto';
 export class MapViewComponent {
     @Input() currentMapView: MapViewInfoDto;
 
+    constructor(private mapViewService: MapViewService) {}
+
     getImage() {
-        return 'http://localhost:3000/api/files/' + this.currentMapView.pictureName;
+        return this.mapViewService.resolvePictureUrl(this.currentMapView);
```
Comment (borys-ihnatyev) [inline src/app/home/map-view/map-view.component.ts:16]: tests will not fail if you remove ```return```

File: src/app/home/map-view/map-view.component.ts [hunk 3]
```diff
@@ -9,7 +10,9 @@ import { MapViewInfoDto } from './map-view.dto';
 export class MapViewComponent {
     @Input() currentMapView: MapViewInfoDto;
 
+    constructor(private mapViewService: MapViewService) {}
+
     getImage() {
```
Comment (borys-ihnatyev) [inline src/app/home/map-view/map-view.component.ts]: Missed type annotation

Comment (piton13) [reply src/app/home/map-view/map-view.component.ts]: fixed.

File: src/app/home/map-view/map-view.component.ts (hunk 2 above)
Comment (piton13) [reply src/app/home/map-view/map-view.component.ts:16]: fixed.

**Pull request merged**